src/composables/useClient.ts

```diff
@@ -55,43 +57,62 @@ export function useClient() {
   }
 
   async function sendEIP712(space, type, payload) {
-    if (type === 'proposal') {
-      let plugins = {};
-      if (Object.keys(payload.metadata?.plugins).length !== 0)
-        plugins = payload.metadata.plugins;
-      return clientEIP712.proposal(auth.web3, web3.value.account, {
-        space: space.id,
-        type: payload.type,
-        title: payload.name,
-        body: payload.body,
-        discussion: payload.discussion,
-        choices: payload.choices,
-        start: payload.start,
-        end: payload.end,
-        snapshot: payload.snapshot,
-        plugins: JSON.stringify(plugins),
-        app: 'snapshot'
-      });
-    } else if (type === 'vote') {
-      return clientEIP712.vote(auth.web3, web3.value.account, {
-        space: space.id,
-        proposal: payload.proposal.id,
-        type: payload.proposal.type,
-        choice: payload.choice,
-        privacy: payload.privacy,
-        app: 'snapshot',
-        reason: payload.reason
-      });
-    } else if (type === 'delete-proposal') {
-      return clientEIP712.cancelProposal(auth.web3, web3.value.account, {
-        space: space.id,
-        proposal: payload.proposal.id
-      });
-    } else if (type === 'settings') {
-      return clientEIP712.space(auth.web3, web3.value.account, {
-        space: space.id,
-        settings: JSON.stringify(payload)
-      });
+    const aliased = space.voting.aliased;
+    try {
+      if (aliased) {
```

Comment:
We should allow alias actions only for these actions: "proposal", "vote" and "delete-proposal"

Reply:
Updated in the latest commit and explained in comments :pray: